src/portal-components/DetailsPage.tsx

```diff
@@ -170,94 +148,110 @@ export default class DetailsPage extends React.Component<
     if (tabLayout?.length) {
       return (
         <div className="DetailsPage tab-layout">
-          <div className="component-container" ref={this.ref}>
+          <div className="component-container">
             {
               <DetailsPageTabs
                 tabConfigs={tabLayout}
                 loading={isLoading}
                 queryResultBundle={this.state.queryResultBundle}
+                showMenu={showMenu}
               ></DetailsPageTabs>
             }
           </div>
         </div>
       )
     } else {
-      const synapseConfigContent = (
```

Comment:
DetailsPageSynapseConfigArray now handles menu (if showMenu is true), so moved this logic